cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -366,7 +370,8 @@ private ActionModellingResult compileAction(Map<String, Object> actionRawData) {
 
         String errorMessagePrefix = "Action syntax is illegal.\n";
         actionData.putAll(
-                transformersHandler.runTransformers(actionRawData, actionTransformers, errors, errorMessagePrefix));
+                transformersHandler.runTransformers(actionRawData, actionTransformers, errors, errorMessagePrefix,
+                        SensitivityLevel.ENCRYPTED));
```

Comment:
@gituser173  what is the resolution here? IMHO you should use the dynamic param to make sure senstivity level is uniform